libraries/api-extractor/src/DocItemLoader.ts

```diff
@@ -1,11 +1,13 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+/* tslint:disable:no-trailing-whitespace */ /* Remove this when GCB-TS is published and upgraded */
```

Comment:
Same reason as your other comment.

---
In reply to: [129396045](https://github.com/Microsoft/web-build-tools/pull/276#discussion_r129396045) [](ancestors = 129396045)